packages/service/src/api/x-sounds/x-sounds-controller.ts

```diff
@@ -22,6 +22,25 @@ class XSoundsController {
     return { sounds };
   }
 
+  @Get('/tags')
+  @ContentType('application/json')
+  async getXSoundsAllTags(): Promise<XSoundsTagsResponseDto> {
+    const tags = await this.xSoundsService.getAllUniqueTags();
+    return { tags };
+  }
+
+  @Get('/tags/:tag')
```

Comment:
Skoro to filtrowanie to powinen być query param do `/api/x-sounds`

https://allegro-restapi-guideline.readthedocs.io/en/latest/Representation/#filtering

`/api/x-sounds?tag=some-tag`